elements/lisk-blocks/package.json

```diff
@@ -30,7 +30,7 @@
 		"uglify:browsertest": "uglifyjs -o ./browsertest.build/browsertest.min.js ./browsertest.build/browsertest.js",
 		"clean": "./scripts/clean.sh",
 		"format": "prettier --write \"*.{ts,js,json}\" \"{src,test}/**/*.{ts,js,json}\"",
-		"lint": "tslint --format verbose --project .",
+		"lint": "echo 'temporally disabled while converting to TS'",
```

Comment:
Maybe be more informative about this by changing the string to lisk-blocks linting temporally disabled while converting to TS

Reply:
same as above